client/src/components/DataFiles/DataFilesModals/DataFilesLinkModal.module.scss

```
@@ -0,0 +1,7 @@
+.action-root {
+  margin-right: 0.5em;
+}
+
+.controls {
+  margin-top: 0.5em;
+}
```

I think you can commit this quick Lint fix directly via an action on this comment.

```suggestion
}

```